src/components/cta/styles.ts

```diff
@@ -0,0 +1,5 @@
+import { cva } from "class-variance-authority";
+
+const ctaStyles = cva("");
```

Comment:
prevediamo di avere delle varianti in futuro? se è vuoto di default forse lo togliere, cosa ne pensate?